catalog/controller/custom/api.php

```diff
@@ -542,4 +541,158 @@ public function lengthCLass()
             $this->setResponseData($query->rows);
         }
     }
+
+    /**
+     * Product Create
+     */
+    public function createProduct()
+    {
+        if (!$this->auth()) {
+            return false;
+        }
+
+        $data = $this->request->post;
+        $this->db->query(
+            "INSERT INTO " . DB_PREFIX . "product SET model = '" . $this->db->escape($data['model']) .
```

Comment:
@byldrm Sürekli aç kapa yapmışız burada, onun yerine tek tırnak veya <<< işaretini kullansak nasıl olur?

Reply:
$query = <<<HEREDOC
This is a $degisken string.

Newlines and everything else is preserved.
HEREDOC;

Böyle bir kullanım var bro. Daha temiz yazabiliriz kodu bu şekilde